src/Status.php

```diff
@@ -154,4 +157,40 @@ public static function getStatusText(int $statusCode): string
             default => ''
         };
     }
+
+    /**
+     * Get a list of all the class constants defined
+     *
+     */
+    static public function getKeyValues(): array
+    {
+        $calledClass = get_called_class();
+
+        try
+        {
+            $oClass = new ReflectionClass($calledClass);
```

Comment:
I hate calling this an oClass.

```suggestion
$pClass = new ReflectionClass($calledClass);
```